### PR TITLE
Test PR review

### PR DESCRIPTION
**Pull request by mondokm**
Test PR

### CODE REVIEW

File: src/main/java/hu/bme/mit/spaceship/GT4500.java [hunk 1]
```diff
@@ -17,6 +17,7 @@ public GT4500() {
 
   public boolean fireLaser(FiringMode firingMode) {
     // TODO not implemented yet
+    System.out.println("Not implemented yet.");
```
Comment (bzp99) [inline src/main/java/hu/bme/mit/spaceship/GT4500.java]: Cool, cool, but please actually implement it

**Review by bzp99 — CHANGES_REQUESTED** [review]
Nice work, but please see my comments in `GT4500.java`

File: src/main/java/hu/bme/mit/spaceship/GT4500.java (hunk 1 above)
Comment (mondokm) [reply src/main/java/hu/bme/mit/spaceship/GT4500.java]: No

Comment (mondokm) [reply src/main/java/hu/bme/mit/spaceship/GT4500.java]: Ok, I did it, please check

**Review by bzp99 — APPROVED** [review]
This is the way, thanks

**Pull request merged**